src/services/root.js

```diff
@@ -32,7 +34,8 @@ module.exports = class RootService {
       interval,
       payload: { watchID: insertedId }
     })
-    if (status < 200 || status >= 300) return Promise.reject("Unable to add this watch to the scheduler")
+    if (status < 200 || status >= 300)
+      return Promise.reject('Unable to add this watch to the scheduler')
```

Comment:
T có nhớ là t bảo ông có thể làm thế này, nhưng mà đây là hàm async/await nên cứ ném thẳng lỗi ra cho lành:
```suggestion
      throw new Error('Unable to add this watch to the scheduler')
```